### ISSUE TITLE
IDEA: Add dropdown to user avatars with two links: 1/ DM (new inbox feature) 2/ Profile 

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/codebuddies#1085]
Idea up for discussion, inspired by feedback @julianjohannesen gave:

"I think it would be helpful to add a drop-down to the letter icon to start a new message and enable folks to search for other users by either name or username"

**Comment by railsstudent** [comment]
@lpatmo good idea.  It is common to group features in a menu under user avatar.  At least, my previous company did that in the web application.
